src/server/components/annual-review/helpers.ts

```diff
@@ -1,18 +1,15 @@
 import { findListById } from "server/models/list";
 import { sendAnnualReviewCompletedEmail } from "server/services/govuk-notify";
 import { lowerCase, startCase } from "lodash";
+import type { User } from "@prisma/client";
 
 export async function sendAnnualReviewCompletedEmailForList(listId: number) {
   const list = await findListById(listId);
 
-  if (list?.jsonData?.users) {
+  if (list?.users) {
     await Promise.all(
-      list.jsonData.users.map(async (postEmailAddress: string) => {
-        await sendAnnualReviewCompletedEmail(
-          postEmailAddress,
-          lowerCase(startCase(list.type)),
-          list?.country?.name ?? ""
-        );
+      list.users.map(async (user: User) => {
+        await sendAnnualReviewCompletedEmail(user.email, lowerCase(startCase(list.type)), list?.country?.name ?? "");
```

Comment:
problem for another time but I wonder why we do lowerCase(startCase - it gives us the same output 🙂